app/views/scheme/show.scala.html

```diff
@@ -19,11 +20,22 @@
     } else {
       </p>
     }
-    @if(! scheme.validator.isEmpty) {
-       <p>Check Topic: <form action="@routes.Application.topicValidate(scheme.id)" method="POST">
-          <input type="text" name="topic">
-          <input type="submit" value="Check">
-       </p>
+    @if(scheme.validator.isDefined) {
```

Comment:
:+1: for switching to a positive condition rather than negative. The code reads so much easier this way.